### ISSUE TITLE
Comprehend and Seek Convolutional Neural Networks over Tree Structures and Building Program Vector Representations for Deep Learning

### ISSUE THREAD

**Issue by Zermy** [sagr4019/ResearchProject#119]
Read the papers again and transfer the used method for vector/AST representation to our current task.

(Convolutional Neural Networks over Tree Structuresfor Programming Language Processing / Building Program Vector Representations for Deep Learning). #116

**Comment by Zermy** [comment]
In the paper "Building Program Vector Rep. are some unkown methods like [stochastic gradient descent algorithm](http://ruder.io/optimizing-gradient-descent/) and [Markov property](https://en.wikipedia.org/wiki/Markov_property). We should clarify the exact implementation if we need these methods for our project.
